lib/components/user/user-account-screen.tsx

```diff
@@ -106,36 +106,19 @@ class UserAccountScreen extends Component<Props> {
     t.style.animation = 'dive-in 1s linear infinite'
     const initialCursor = firstLabel ? firstLabel.style.cursor : ''
     if (firstLabel) firstLabel.style.cursor = 'wait'
-    const loadingToast = !isWizard
-      ? toast.loading(
-          intl.formatMessage({ id: 'components.UserAccountScreen.updating' })
-        )
-      : undefined
+    const submitSuccessMessage = formattedToastSuccessMessage(
+      intl.formatMessage({
+        // Use a summary text for the field, if defined (e.g. to replace long labels),
+        // otherwise, fall back on the first label of the input.
+        defaultMessage: firstLabel?.innerText,
+        id: `components.UserAccountScreen.fields.${t.name}`
+      }),
+      intl.formatMessage({
+        id: 'components.UserAccountScreen.fieldUpdated'
+      })
+    )
     try {
-      await submitForm()
-      // On success, display a toast notification for existing accounts.
-      if (!isWizard) {
-        toastSuccess(
-          intl.formatMessage({
-            // Use a summary text for the field, if defined (e.g. to replace long labels),
-            // otherwise, fall back on the first label of the input.
-            defaultMessage: firstLabel?.innerText,
-            id: `components.UserAccountScreen.fields.${t.name}`
-          }),
-          intl.formatMessage({
-            id: 'components.UserAccountScreen.fieldUpdated'
-          }),
-          loadingToast
-        )
-      }
-    } catch {
-      // Remove any toasts before showing alert.
-      toast.remove()
-      alert(
-        intl.formatMessage({
-          id: 'components.UserAccountScreen.errorUpdatingProfile'
-        })
-      )
+      await toastPromise(submitForm(), submitSuccessMessage, intl)
```

Comment:
In "wizard" mode (e.g. when creating accounts), don't show toasts in response to individual field changes. Otherwise, each change will be toasted/announced in addition to the final toast when the user clicks "Finish".